release.md

```diff
@@ -0,0 +1,173 @@
+  * [Metadata for releasing](#metadata-for-releasing)
+  * [Versioning](#versioning)
+
+[Triggering a Release](#)
+
+[FAQ](#faq)
+
+## General overview
+
+Ignition development supports different ways of installing the software in the
+user system. This document covers the process of creating binaries for different
+distributions starting from a local copy of source code together with
+descriptions and explanations of preparing, creating and releasing
+new software versions.
+
+![general releasing context](releasing/images/general_context.png)
+
+### Releasing platforms supported
+
+The Ignition software supports a large variety of platforms where it can be
+built from source. A small subgroup of them are selected to be officially
+supported for binary distribution:
+
+ * **Linux:** .deb packages are being created for different Ubuntu and Debian
+   releases. Ubuntu LTS and Debian stable are officially supported.
+   Architectures supported are typically `amd64`, `arm64` and `armhf` but that
+   combination may not be available for all Ubuntu/Debian/Ignition releases.
+
+ * **macOS:** binaries for all active releases of macOS are usually created by
+   the Ignition development team under the support of
+   [Homebrew package manager](https://brew.sh/). Binary packages for Brew are
+   called `bottles`.
+
+ * **Windows:** currently *there is no binary distribution for Windows* that covers
+   all Ignition libraries although there is an ongoing effort for getting
+   binary packages inside
+   [conda-forge](https://github.com/conda-forge/staged-recipes/issues/13551)
+
+### Infrastructure
+
+The Ignition team uses different pieces of infrastructure to execute the
+releasing process:
+
+ * **build.osrfoundation.org**: main Jenkins server to orchestrate the whole
+   releasing process. It receives requests from Ignition developers to start the
+   release process, schedule nightly builds, build the binary packages,
+   upload releasing artifacts to repositories, etc.
+
+ * **packages.osrfoundation.org**: Debian/Ubuntu .deb repositories. The URLs
+   are ready to be used by the APT packages system.
+
+ * **osrf-distributions S3**: [cloud storage](http://gazebosim.org/distributions/bottles-simulation/)
+   space to save the different artifacts produced during the release process. The
+   S3 space is no more than archive of artifacts that need to be manually
+   downloaded (i.e: to get an old version of a .deb package) or handled by other
+   packaging systems (i.e: brew download the bottles from S3, the buildfarm uses
+   the tarballs stored in S3 when building .deb packages)
+
+### Type of releases
+
+The Ignition development uses three different types of releases depending on the
+stability of the software:
+
+ * **stable**: stable releases are designed to be generally consumed by users
+   that do not expect to solve severe issues or regressions.
+   * Linux stable binaries:
+     http://packages.osrfoundation.org/gazebo/{ubuntu/debian}-stable
```

Comment:
What is the `{ubuntu/debian}` syntax in the URL? Does that mean users should pick one or the other? If so, perhaps we should explicitly mention this.

Reply:
I tried here ab05d29f454824b8ba94a35e77d3578366746d3a